civictechprojects/views.py

```diff
@@ -18,20 +18,27 @@
 from democracylab.models import Contributor, get_request_contributor
 from common.models.tags import Tag
 
-from pprint import pprint
-
 def tags(request):
     url_parts = request.GET.urlencode()
     query_terms = urlparse.parse_qs(
         url_parts, keep_blank_values=0, strict_parsing=0)
     if 'category' in query_terms:
```

Comment:
Add a TODO for adding a flag to add the tag counts, as there are some instances (like populating the project tag dropdowns) where we don't need the tag counts.